ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -72,6 +72,11 @@ message Commands {
     // Must be non-negative. Must not exceed the maximum deduplication time (see
     // ``ledger_configuration_service.proto``).
     google.protobuf.Duration deduplication_duration = 15;
+
+    // Specifies the start of the deduplication period by a completion stream offset.
+    //
+    // Must be a valid LedgerString (as described in ``ledger_offset.proto``).
```

Comment:
@andreaslochbihler-da: I remember a message of yours saying that empty lines in these field comments are problematic, are they?

Reply:
They are problematic for our protobuf docs template. The comment was by @cocreature: https://github.com/digital-asset/daml/pull/11887#issue-1063870449